src/worker.js

```diff
@@ -1,11 +1,35 @@
 'use strict';
 
+// Famous dependencies
 var DOMElement = require('famous/dom-renderables/DOMElement');
 var FamousEngine = require('famous/core/FamousEngine');
 
-// App Code
-var scene = FamousEngine.createScene();
-var root = scene.addChild();
-var el = new DOMElement(root);
-el.setProperty('background', 'yellow');
+// Initialize with a scene; then, add a 'node' to the scene root
+var logo = FamousEngine.createScene().addChild();
```

Comment:
In the web worker version, why didn't call  FamousEngine.setChannel?
Without a valid Channel, then how "engine" communicate to the renderers"?